charts/kong/templates/_helpers.tpl

```diff
@@ -1675,6 +1674,13 @@ Kubernetes Cluster-scoped resources it uses to build Kong configuration.
   verbs:
   - get
   - update
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
+  verbs:
+  - get
+  - list
```

Comment:
```suggestion
  - list
  - watch
```

Reply:
Why would we need the `watch` 🤔 ? This is to address the lookup in  https://github.com/Kong/kubernetes-ingress-controller/blob/2889365e4908b7aa49fbb07ab783cfdf8ff72ff3/internal/controllers/gateway/route_utils.go#L454-L458 (which uses the cached client hence the need for `list`).

Reply:
It's because the underlying client uses `watch` to keep the cache in sync apparently according to this: https://github.com/kubernetes-sigs/controller-runtime/issues/1156

Reply:
But that's only for the resources that we watch using `For()` or `Owns()`, no? Just like the linked issue suggests

Reply:
I understand `For()` and `Owns()` is just another example when `watch` is required. The issue itself is purely about `watch` being required when `client.Get()` is used. 🤔

Reply:
So I tried to reproduce the original issue and it seems that we do need the `watch`.

Without it I keep getting 

```
E1212 16:51:09.392184       1 reflector.go:147] pkg/mod/k8s.io/client-go@v0.28.3/tools/cache/reflector.go:229: Failed to watch *v1.Namespace: unknown (get namespaces)
```

Let me add that and mention it in the changelog.

Good call!